mqemitter-redis.js

```diff
@@ -91,7 +91,7 @@ MQEmitterRedis.prototype.close = function (cb) {
 }
 
 MQEmitterRedis.prototype._subTopic = function (topic) {
-  return topic.replace(this._opts.wildcardOne, '*')
+  return topic.replace(new RegExp(this._opts.wildcardOne.replace(/([/,!\\^${}[\]().*+?|<>\-&])/g, '\\$&'), 'g'), '*')
```

Comment:
can you avoid instantiating a new RegExp for every call to `_subTopic`? This is going to be slow.

Reply:
Thanks for the review. Yes, of course. I am aware of the performance issue.  I did in the first version (in the constructor/lazy init in the method) but I've noticed that you can change the symbol after instantation ( a unit test failed ). I will intercept the setter call and replace the regex with the new one (if the symbol will change obviously). I will do it at the beginning of the next week.

Reply:
I remembered a false fact 

>  I've noticed that you can change the symbol after instantation ( a unit test failed )

Now RegExp is instantiated in the constructor and tests pass without any issue.